### ISSUE TITLE
(after release) Move Homebrew formula to the main Homebrew repository

### ISSUE THREAD

**Issue by cweagans** [universal-ctags/ctags#658]
If we move to the main homebrew repo, then their build farm will take care of building binaries for 4 different versions of Mac OS. I'll take care of this.

**Comment by cicloid** [comment]
Do you need help with this?

**Comment by kopischke** [comment]
Good idea. Note however that I have found that the formula does not match the current build process and prerequisites. I am maintaining an updated one [here](https://github.com/kopischke/homebrew-ctags) which I would be happy to contribute if you find it useful and tell me how (can’t do a PR as it is not a fork of the official repo).

**Comment by cweagans** [comment]
Yes, please do contribute it. If you fork the official repo and merge in your changes manually, you can open a PR, tag me in it, and I will merge it.

**Comment by ivanbrennan** [comment]
It would great if we had a tagged release so this could move forward. I made a PR to expose the `--with-universal-ctags` option in the [GNU global](https://www.gnu.org/software/global/manual/global.html#Plug_002din) homebrew [formula](https://github.com/Homebrew/homebrew-core/pull/8309), but it can't go forward while the universal ctags formula lies outside the homebrew-core repository.

https://github.com/Homebrew/homebrew-core/pull/8309

**Comment by ygini** [comment]
Please, could you consider a MacPort version for Mac? MacPort is way more suitable for enterprise deployment (multi user for real, no permission issues, able to create a standalone package from the port tree to avoid using MacPort on all devices except the admin engine, etc.).

In all way, it's way more than better to use MacPort instead of Homebrew, and the work done for MacPort will bring you closer to a support for BSD ports.
